packages/sage/src/commands/upgrade.ts

```diff
@@ -14,6 +14,7 @@ import { Config, Plugin } from 'robo.js'
 
 const command = new Command('upgrade')
 	.description('Upgrades your Robo to the latest version')
+	.option('-y --auto-accept', 'installs updates without showing changelogs')
```

Comment:
`-y` as the short form of `--auto-accept` feels kinda weird here as those words are completely different. Please name it `--yes` for consistency.

Reply:
owo